docs/run-platform-server/installing.mdx

```diff
@@ -0,0 +1,44 @@
+---
+title: Installing
+sidebar_position: 20
+---
+
+import { CodeExample } from "@site/src/components/CodeExample";
+
+To install Horizon, you have choices, we recommend the following for target infrastructure:
+
+- bare-metal:
+  - if host is debian linux, install prebuilt binaries [from repositories](#package-manager) using package manager.
+  - for any other hosts, download [prebuilt release binaries](#prebuilt-releases) of Stellar Horizon and Core for host target architecture and operation system.
+- containerized:
+  - non-Helm Chart, if the target envrironment for container to run does not support Helm chart usage, run the prebuilt docker image of Horizon published on [dockerhub.com/stellar/horizon](https://hub.docker.com/r/stellar/stellar-horizon).
+  - Helm charts, when the target envrionment uses container orchestration such as Kubernetes and has enabled Helm Charts on cluster. The Horizon Helm chart manages installation life cycle. Use the [Helm Install command](https://helm.sh/docs/helm/helm_install/), it will accept Horizon's configuration parameters. Please review [Configuration](./configuring.mdx) first, to identify any specific configuration params needed. **TODO - create the Horizon Helm Chart.**
```

Comment:
Hello @jacekn , as part of new documentation, we are referring to the potential Helm Chart usage for Horizon. Wondering if we shouild spin up a dev ticket for Horizon Helm Chart on the mono repo, or if you are aware of Helm for Horizon already, should we coordinate with an Ops ticket for this feature? CC'ing @urvisavla